src/overlays/components/OverlayController.tsx

```diff
@@ -36,6 +36,7 @@ import AutoRefreshOverlay from 'src/dashboards/components/AutoRefreshOverlay'
 import CellCloneOverlay from 'src/shared/components/cells/CellCloneOverlay'
 import CustomApiTokenOverlay from 'src/authorizations/components/redesigned/CustomApiTokenOverlay'
 import DisplayTokenOverlay from 'src/authorizations/components/redesigned/DisplayTokenOverlay'
+import CreateSecretOverlay from 'src/shared/components/secrets/CreateSecretOverlay'
```

Comment:
the rest of these (with the exception of cell cone for some reason) are all scoped to their functional folders, why move this stuff out of the existing `src/secrets` folder into a shared folder?

Reply:
done